enterprise/backend/test/metabase_enterprise/metabot_v3/models/metabot_use_case_test.clj

```diff
@@ -7,21 +7,17 @@
 
 (use-fixtures :once (fixtures/initialize :db))
 
-(deftest resolve-profile-test
+(deftest fetch-use-case-test
   (mt/with-temp [:model/Metabot {metabot-id :id} {:name "Test Metabot"}
                  :model/MetabotUseCase _ {:metabot_id metabot-id :name "nlq" :profile "internal" :enabled true}
                  :model/MetabotUseCase _ {:metabot_id metabot-id :name "sql" :profile "internal" :enabled true}
                  :model/MetabotUseCase _ {:metabot_id metabot-id :name "transforms" :profile "transforms_codegen" :enabled true}
-                 :model/MetabotUseCase _ {:metabot_id metabot-id :name "omnibot" :profile "internal" :enabled true}]
-    (testing "resolves profile from use case in DB"
-      (is (= "internal" (metabot-v3.config/resolve-profile metabot-id "nlq" nil)))
-      (is (= "internal" (metabot-v3.config/resolve-profile metabot-id "sql" nil)))
-      (is (= "transforms_codegen" (metabot-v3.config/resolve-profile metabot-id "transforms" nil)))
-      (is (= "internal" (metabot-v3.config/resolve-profile metabot-id "omnibot" nil))))
-
-    (testing "profile override takes precedence"
-      (is (= "custom-profile" (metabot-v3.config/resolve-profile metabot-id "nlq" "custom-profile")))
-      (is (= "custom-profile" (metabot-v3.config/resolve-profile metabot-id "transforms" "custom-profile"))))
+                 :model/MetabotUseCase _ {:metabot_id metabot-id :name "omnibot" :profile "internal" :enabled false}]
+    (testing "fetches use case from DB"
+      (is (= {:profile "internal" :enabled true} (metabot-v3.config/fetch-use-case metabot-id "nlq")))
+      (is (= {:profile "internal" :enabled true} (metabot-v3.config/fetch-use-case metabot-id "sql")))
```

Comment:
I'm not sure yet what profile names we want to standardize on for these - so for now I've defaulted them both to `internal` to retain the current behavior. Once the inline SQL metabot is merged I'll have a follow-up PR to integrate with it and change these to their final values.